highfidelity/hifi
# MS17774: Enable advanced movement on first run

- Implementation of MS[#17774](https://highfidelity.manuscript.com/f/cases/17774)

**TEST PLAN**
- Launch Interface
- In `Settings > Controls`, "Advanced movement for VR" option should be checked
- Switch to HMD mode
- Moving with the left thumbpad / joystick to move around should work
